tests/gumjs/script.c

```diff
@@ -6174,6 +6263,36 @@ TESTCASE (memory_can_be_scanned_synchronously)
   EXPECT_SEND_MESSAGE_WITH ("\"done\"");
 }
 
+TESTCASE (memory_can_be_scanned_asynchronously)
+{
+  guint8 haystack[] = { 0x01, 0x02, 0x13, 0x37, 0x03, 0x13, 0x37 };
+
+  COMPILE_AND_LOAD_SCRIPT (
+      "Promise.resolve(Memory.scan(" GUM_PTR_CONST ", 7, '13 37', {"
+      "  onMatch(address, size) {"
+      "    send('onMatch offset=' + address.sub(" GUM_PTR_CONST ").toInt32()"
+      "      + ' size=' + size);"
+      "  }"
+      "})).catch(e => console.error(e.message)).then(() => send('DONE'));", haystack, haystack);
```

Comment:
This exceeds 80 cols. (Would be good to also verify that no other lines exceed 80 cols.)

Reply:
Oops, sorry for the rookie mistake! Done.